includes/libsUtils.hpp

```diff
@@ -62,6 +64,10 @@
 // # define RPL_AWAY(nickname, username, hostname, awayMessage) USER_ID(nickname, username, hostname) + " " + (nickname) + " :" + (awayMessage)
 // # define RPL_CHANNEL_MODE_IS(nickname, username, hostname, channel, mode, modeParams) USER_ID(nickname, username, hostname) + " " + (channel) + " " + (mode) + " " + (modeParams)
 # define RPL_END_OF_NAMES(nickname, usermane, hostname, channel) USER_ID(nickname, username, hostname) + " " + (channel) + " :End of NAMES list."
+# define RPL_WELCOME(nickname, username, hostname) std::string("Welcome to the Internet Relay Network ") + USER_ID(nickname, username, hostname)
+# define RPL_YOURHOST(servername)"Your host is " + servername + ", running version ft_messenger-v1.0.0"
```

Comment:
```suggestion
# define RPL_YOURHOST(servername) "Your host is " + servername + ", running version ft_messenger-v1.0.0"
```

Reply:
`# define RPL_YOURHOST(servername) "Your host is " + (servername) + ", running version ft_messenger-v1.0.0"`

Por otro lado, pondría la version en otro define, por si la cambiamos en un futuro